jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -8325,13 +8325,145 @@ public String indent(int n) {
 /*[ENDIF] Java12 */
 
 /*[IF Java13]*/
-	/*
+	/**
 	 * Determine if current String object is LATIN1.
 	 *
 	 * @return true if it is LATIN1, otherwise false.
 	 */
 	boolean isLatin1() {
 		return LATIN1 == coder();
 	}
+
+	/**
+	 * Format the string using this string as the format with supplied args.
+	 *
+	 * @param args
+	 *          the format arguments to use
+	 *
+	 * @return the formatted result
+	 *
+	 * @see #format(String, Object...)
+	 */
+	@Deprecated(forRemoval=true, since="13")
+	public String formatted(Object... args) {
+		return String.format(this, args);
+	}
+
+	/**
+	 * Removes the minimum indentation from the beginning of each line and
+	 * removes the trailing spaces in every line from the string
+	 *
+	 * @return this string with incidental whitespaces removed from every line
+	 */
+	@Deprecated(forRemoval=true, since="13")
+	public String stripIndent() {
+		Iterator<String> iter = lines().iterator();
+		StringBuilder builder = new StringBuilder();
+		int min = Integer.MAX_VALUE;
+
+		while (iter.hasNext()) {
+			String line = iter.next();
+
+			if ((!line.isBlank()) || (!iter.hasNext())) {
+				int count = 0;
+				while ((count < line.length())
+					&& (Character.isWhitespace(line.charAt(count))
+					&& (count < min))
+				) {
+					count++;
+				}
+				min = (count < min ? count : min);
+			}
+
+			if (line.isBlank()) {
+				builder.append("");
+			} else {
+				builder.append(line.stripTrailing());
+			}
+			if (iter.hasNext()) {
+				builder.append("\n");
+			}
+		}
+
+		return builder.toString().indent(-min);
+	}
+
+	/**
+	 * Translate the escape sequences in this string as if in a string literal
+	 *
+	 * @return result string after translation
+	 *
+	 * @throws IllegalArgumentException
+	 *          If invalid escape sequence is detected
+	 */
+	@Deprecated(forRemoval=true, since="13")
+	public String translateEscapes() {
+		StringBuilder builder = new StringBuilder();
+		char[] charArray = toCharArray();
+		int index = 0;
+		int strLength = length();
+		while (index < strLength) {
+			if (charArray[index] == '\\') {
+				if (index < strLength - 1) {
```

Comment:
I think this condition needs to be inverted. If you move the increment below before this test it can be simplified to `index >= strLength`.

Reply:
updated as suggested